### PR TITLE
Crossref fix

### PR DESCRIPTION
**Pull request by bondjimbond**
Fixes Travis issue mentioned in #4 

@DiegoPino

### CODE REVIEW

**Comment by DiegoPino** [comment]
@bondjimbond 
see https://travis-ci.org/Islandora/islandora_badges/jobs/272963922#L703-L721
some indenting issues also?

**Review by whikloj — CHANGES_REQUESTED** [review]
Hey @bondjimbond 
Travis still has a problem [here](https://travis-ci.org/Islandora/islandora_badges/jobs/275159469#L763)

You can run the various test commands locally before pushing it up to ensure you've got them all.
There are listed here - https://github.com/Islandora/islandora_badges/blob/7.x/.travis.yml#L35-L40

**Comment by bondjimbond** [comment]
@whikloj Speaking of Travis, shouldn't the directories in https://github.com/Islandora/islandora_badges/blob/7.x/.travis.yml#L34-L35 and https://github.com/Islandora/islandora_badges/blob/7.x/.travis.yml#L24 be changed to /islandora_badges instead of /islandora_altmetrics?

**Comment by whikloj** [comment]
@bondjimbond for consistency yes, but as those lines are just referencing the link created here (https://github.com/Islandora/islandora_badges/blob/7.x/.travis.yml#L24) it doesn't really matter. We can get @DiegoPino to change those once we have a green light ready to go.

**Comment by bondjimbond** [comment]
@whikloj @DiegoPino Looks like coding standards are fine, but travis.yml hasn't been updated for the php 5.3.3 issue.

File: modules/islandora_crossref_citations/islandora_crossref_citations.module
Comment (whikloj) [inline modules/islandora_crossref_citations/islandora_crossref_citations.module]: @bondjimbond Not that I want to turn this into a huge thing, but perhaps you should use `format_string()` here to avoid the multiple concatenations? 
Something like...
```
$badge = format_string('<img src="https://img.shields.io/badge/!block_text-!result_citation_count-blue.svg?style=flat" alt="Number of citations via Crossref: !result_citation_count">',
  array(
    '!block_text' => $block_text,
    '!result_citation_count' => $result_citation_count,
  )
);
```

Comment (bondjimbond) [reply modules/islandora_crossref_citations/islandora_crossref_citations.module]: Sure, that seems to work OK.

**Comment by whikloj** [comment]
Thanks @bondjimbond, did you test my suggestion? I just made it up off the top of my head. Just wondering if it does work?

**Comment by bondjimbond** [comment]
@whikloj Yes, tested and working - it's in the latest commit here.

**Review by whikloj — APPROVED** [review]

**Comment by whikloj** [comment]
@DiegoPino can you rebase #4 on this and see if it helps.

**Comment by bondjimbond** [comment]
@whikloj It's also stuck on your first review waiting for approval. I think that was all addressed, right?

**Comment by DiegoPino** [comment]
@whikloj thanks!!

**Pull request merged**
